ffmpeg/api_test.go

```diff
@@ -1322,6 +1322,46 @@ func TestTranscoderAPI_ClipInvalidConfig(t *testing.T) {
 	}
 }
 
+func noKeyframeSegment(t *testing.T, accel Acceleration) {
+	// Reproducing #219
+	run, dir := setupTest(t)
+	defer os.RemoveAll(dir)
+
+	cmd := `
+		cp "$1"/../transcoder/kryp-*.ts .
```

Comment:
We should move these segments to `data` instead of `transcoder` directory, and update this line, as we now store all test segments there.